mail/views_test.py

```diff
@@ -636,3 +641,89 @@ def test_send_financial_aid_view_improperly_configured(self):
         with patch('mail.views.MailgunClient.send_batch', side_effect=ImproperlyConfigured):
             resp = self.client.post(self.url, data=self.request_data, format='json')
         assert resp.status_code == status.HTTP_500_INTERNAL_SERVER_ERROR
+
+
+@ddt.ddt
+class EmailBouncedViewTests(APITestCase, MockedESTestCase):
+    """Test email bounce view web hook"""
+    @classmethod
+    def setUpTestData(cls):
+        super().setUpTestData()
+        cls.url = reverse('email_bounced_view')
```

Comment:
`setUpTestData` should only be used for setting up database objects which are reused for each test. If you just need the string value set this in `setUp` instead.